pkg/restapi/operation/operations.go

```diff
@@ -58,6 +59,17 @@ type Handler interface {
 	Handle() http.HandlerFunc
 }
 
+// UUIDBuilder represents a type that can generate a uuid string
+type UUIDBuilder interface {
+	New() uuid.UUID
+}
+
+type uuidType struct{}
```

Comment:
Generally you want to avoid using generic self-describing words like `type` as part of the type name... (i.e. uuidType is a type, so you don't need to also call the type a `type`)

perhaps you can name it uuidBuilder or uuidGenerator or something similar.

Reply:
fixed